requirements.txt

```diff
@@ -0,0 +1,5 @@
+pip==24.2
+psutil==6.0.0

```

Comment:
Why do we need to distribute `psutil` with pythia ?

For the long term, my expectation is that pythia, will be just `python + embedded pip` 

No other requirements.

setuptools / pywin32 / pycparser can all be installed later in chevah/server and chevah/compat via `pip` from our PyPI mirror

Reply:
It's been added as requested through https://github.com/chevah/python-package/pull/125.

It can also be removed as requested. I've added a ticket to deal with all this: https://github.com/chevah/pythia/issues/66

Reply:
Note though that `psutil` doesn't have a musl wheel, so we need to build it until upstream changes its mind. More at https://github.com/giampaolo/psutil/pull/2126

Reply:
thanks. That's ok. We just need to add a comment that `psutils` is still needed and link to the upstream bug report.

In theory, on alpine, we can build the wheel for psutil and then copy it to bin.chevah.com .. it will be available as wheel for Alpine just for us